src/Stepper/Step.less

```diff
@@ -2,16 +2,16 @@
 @import (reference) "../less/index";
 
 @stepIconWidth: 1.75rem; // 28px
-@iconBorderWidth: @size_4xs; // 1px
 @iconFocusShadowThickness: @size_2xs; // 4px
 @titleFontSize: 1.25rem; // 20px
 @alertSize: 0.375rem; // 6px
 
 .Step {
   .flexbox();
   .alignItems(flex-start);
-  width: 100%;
   .padding--y--2xs;
+  .text--medium;
+  .text--regular;
 }
 
 .Step--button {
```

Comment:
can this be consolidated with `.Step`? couldn't tell why they're separate.

Reply:
not sure what you mean by consolidated.

Reply:
not totally. .button--reset() cannot be used when Step is a <div> because it changes the pointer.